packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -69,46 +69,6 @@ describe('parsing', () => {
     });
 });
 
-describe('class and style', () => {
```

Comment:
I don't see any value in keeping those tests around. After the AST conversion, this test suite will not be needed anymore.